docs/swagger.yaml

```diff
@@ -49,8 +120,10 @@ paths:
           description: Successfully retrieved the CustomerData object for the given session id.
         '204':
           description: 'The specified session id is recognized, but no data is available yet (login flow not yet completed by user).'
```

Comment:
These responses no longer corresponds to what the system actually does, and should be updated.

Reply:
Fixed it